v3/src/components/graph/components/parent-toggles.tsx

```diff
@@ -74,11 +77,14 @@ export const ParentToggles = observer(function ParentToggles() {
   const { tileWidth } = useGraphLayoutContext()
   const graphModel = useGraphContentModelContext()
   const dataConfig = useGraphDataConfigurationContext()
+  const primaryAttribute = dataConfig?.primaryAttributeID ?? ""
   const dataset = dataConfig?.dataset
+  const collectionIndexForPrimaryAttribute = dataset?.getCollectionIndexForAttribute(primaryAttribute) ?? 0
   const isCollectionSet = !!(dataset && dataset.collections?.length > 0)
   const hiddenCases = dataConfig?.hiddenCases ?? []
-  const caseIDs = dataset?.items.map((c) => c.__id__) ?? []
-  const caseButtons = createCaseButtons({ caseIDs, dataset, isCollectionSet, hiddenCases })
+  const itemIDs = dataset?.items.map((c) => c.__id__) ?? []
```

Comment:
The DataSet has an `itemIds` method, so you should just be able to use:
```suggestion
  const itemIDs = dataset?.itemIds ?? []
```